istioctl/cmd/kubeinject.go

```diff
@@ -367,7 +529,8 @@ kube-inject on deployments to get the most up-to-date changes.
 		fmt.Sprintf("ConfigMap name for Istio mesh configuration, key should be %q", configMapKey))
```

Comment:
Can we deprecate the parameters as John suggested, so we can remove them in the future?

Reply:
fixed